site/_data/eleventyComputed.js

```diff
@@ -305,13 +305,25 @@ module.exports = {
         .sort(propSort({ prop: 'data.title', lowercase: true }))
         .sort(propSort({ prop: 'data.weight', fallback: 0 }))
         .map((i) => ({
+          eyebrow: get(i, 'data.tags[0]'),
           title: get(i, 'data.title'),
           image: get(i, 'data.app.logo'),
+          logo: {
+            src: get(i, 'data.app.logo'),
+            name: get(i, 'data.app.name'),
+            company: get(i, 'data.app.company'),
+          },
+          media: {
+            src: get(i, 'data.hero.image'),
+            alt: get(i, 'data.hero.alt'),
+          },
```

Comment:
Media should pull from `data.featured` because items that get featured may have different featuring images than hero images that are used in the post itself. I believe we already have computed data to uplevel `hero` into `featured` if there isn't a featured image